src/component/molecules/navbarMenuRight/navbarMenuCustomize.tsx

```diff
@@ -0,0 +1,12 @@
+import { Badge, Box } from "@mui/material";
+import styled from "styled-components";
+
+export const CustomizeBadge = styled(Badge)`
```

Comment:
Be consist. Sometimes you have `NameCustomize` and sometimes you have `CustomizeName`